packages/react-composites/src/composites/CallComposite/Strings.tsx

```diff
@@ -322,4 +322,11 @@ export interface CallCompositeStrings {
    * Control bar People button ToolTipContent
    */
   peopleButtonTooltipClose: string;
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+  /**
+   * label for when making a outbound call
+   *
+   * Will either be the title or the description based on the type of call you are making.
```

Comment:
I think this comment adds confusion. Maybe it's better to omit this line.
OR change it such that it's easier to understand.